### PR TITLE
clarify dev setup instructions

### PR DESCRIPTION
**Pull request by Almenon**
see commit messages for explanations.

### CODE REVIEW

**Review by kalvdans — DISMISSED** [review]

File: app/readme.md [hunk 1]
```diff
@@ -17,6 +17,9 @@ To get the code onto your machine, clone the git respository from GitHub with th
 git clone https://github.com/Couchers-org/couchers.git
 ```
 
+If you use Windows clone the repo using [WSL2](https://docs.microsoft.com/en-us/windows/wsl/install-win10).
+WSL2 lets you easily use docker and linux utilities on Windows. This also avoids a problem with the [dev env on native windows](https://github.com/Couchers-org/couchers/issues/389).
+
 Open a command line or terminal, and navigate to the `app` folder (where this `readme.md` is), for example with `cd couchers/app`.
```
Comment (aapeliv) [inline app/readme.md]: Could we swap the order of these two lines?

Comment (Almenon) [reply app/readme.md]: Done, does it look good? If so please merge in because I will be at work. I'll be back tonight.

Comment (aapeliv) [reply app/readme.md]: Sorry, I meant that you should move the comment about WSL2 under the "Open a command line...".

Did you erroneously add a heading about a prereq step?

Comment (Almenon) [reply app/readme.md]: errr, yeah. Good catch. I've switched the order, look good?

**Comment by Almenon** [comment]
@kalvdans the review type says you left comments but I don't see any, do you have any comments about the PR?

**Review by aapeliv — APPROVED** [review]
Yes, thanks. Feel free to merge.

**Comment by kalvdans** [comment]
> @kalvdans the review type says you left comments but I don't see any, do you have any comments about the PR?

I checked the Approve radio button without writing any message. Seems github hides the approval status once the review is stale, so it ends up empty. So I better write something in the text box next time!

**Review by kalvdans — APPROVED** [review]
LGTM!

**Pull request merged**